src/index.html

```diff
@@ -21,7 +24,322 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="body page__body">
+    <header class="header">
+      <a
+        href="#Home"
+        class="header__img"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo"
+        />
+      </a>
+      <nav class="header__nav">
+        <a
+          href="#Apple"
+          class="header__link is-active"
+        >
+          Apple
+        </a>
+        <a
+          href="#Samsung"
+          class="header__link"
+        >
+          Samsung
+        </a>
+        <a
+          href="#Smartphones"
+          class="header__link"
+        >
+          Smartphones
+        </a>
+        <a
+          href="#Laptops & Computers"
+          class="header__link"
+          data-qa="nav-hover"
```

Comment:
The `data-qa` attribute is used here for the 'Laptops & Computers' link. Ensure that this attribute is applied consistently across all elements that require it for testing or automation purposes.